### PR TITLE
Add store limit guide

### PR DESCRIPTION
**Pull request by rleungx**
### What is changed, added or deleted? (Required)

Add store limit guide for release 3.1 branch.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version) (covered in https://github.com/pingcap/docs-cn/pull/3831)
- [ ] v4.0 (TiDB 4.0 versions) (covered in https://github.com/pingcap/docs-cn/pull/3832)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: configure-store-limit.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Store Limit
+summary: 介绍 Store Limit 功能。
+category: how-to
+---
+
+# Store Limit
+
+Store Limit 是 PD 在 3.0 版本引入的特性，旨在能够更加细粒度地控制调度的速度，针对不同调度场景进行调优。
```
Comment (lhy1024) [inline configure-store-limit.md:9]: 是否需要强调 3.0 和 3.1 中的不同之处？

File: configure-store-limit.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+### 持久化 store limit 修改
+
+由于 store limit 是一个内存中的映射关系，所以上述的修改在切换 leader 或者 PD 重启后会被重置。如果同时想要持久化修改，可以同时使用下面的方法进行设置：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+config set store-balance-rate 20    // 将所有 store 添加 learner/peer 和删除 peer 的速度上限为每分钟 20 个
```
Comment (ran-huang) [inline configure-store-limit.md:78]:
```suggestion
config set store-balance-rate 20    // 将所有 store 添加 learner/peer 和删除 peer 的速度上限为每分钟 20 个。
```

**Comment by ran-huang** [comment]
@rleungx Please add this doc to TOC. Thanks!

**Pull request merged**